run-tests

```diff
@@ -9,8 +9,8 @@ set -u
 # options, or read the output below.
 #
 
-image=w0rp/ale
-current_image_id=f58c7bf8900f
+image=akevinclark/ale
+current_image_id=8e1802ca34b5
```

Comment:
Note that this change likely needs to be reverted back to @w0rp's docker if this is to be merged. But in the meantime I wanted CI to be able to use the new Dockerfile.